src/Components/AddMoviesPopup/AddMoviesPopup.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC, useState } from 'react';
+import { Box } from '../Box/Box';
+import { TopRightButton } from '../Header/Header.styled';
+import { Modal } from '../Modal/Modal';
+
+export const AddMoviesPopup: FC = () => {
+  const [modalOpen, setModalOpen] = useState(false);
+
+  return (
+    <>
+      <TopRightButton right="6" top="6" onClick={() => setModalOpen(true)}>
+        + add movie
+      </TopRightButton>
+      <Modal show={modalOpen} onClose={() => setModalOpen(false)}>
```

Comment:
It's a good example to understand how to use React "useCallback" hook here - `() => setModalOpen(false)`

Reply:
Thanks for the suggestion, code has been updated to use the hook "useCallBack";